frame/vamm/src/tests/create_vamm.rs

```diff
@@ -0,0 +1,159 @@
+use crate::{
+	mock::{Balance, Event, ExtBuilder, MockRuntime, System, TestPallet},
+	pallet::{self, Error, VammMap, VammState},
+	tests::{loop_times, min_max_reserve, zero_reserve, VammTimestamp, RUN_CASES},
+};
+use composable_traits::vamm::{Vamm as VammTrait, VammConfig};
+use frame_support::{assert_noop, assert_ok};
+use proptest::prelude::*;
+
+proptest! {
+	#![proptest_config(ProptestConfig::with_cases(RUN_CASES))]
+	#[test]
+	#[allow(clippy::disallowed_methods)]
+	fn create_vamm(
```

Comment:
I would change name to create_vamm_correctly_updates_runtime_storage

Reply:
Thanks for the input.
I changed it :+1: